target/linux/ar71xx/files/arch/mips/ath79/Makefile

```diff
@@ -198,6 +198,7 @@ obj-$(CONFIG_ATH79_MACH_TL_WR1043ND_V4)		+= mach-tl-wr1043nd-v4.o
 obj-$(CONFIG_ATH79_MACH_TL_WR2543N)		+= mach-tl-wr2543n.o
 obj-$(CONFIG_ATH79_MACH_TL_WR703N)		+= mach-tl-wr703n.o
 obj-$(CONFIG_ATH79_MACH_TL_WR720N_V3)		+= mach-tl-wr720n-v3.o
+obj-$(CONFIG_ATH79_MACH_TL_WR740N_V6)		+= mach-tl-wr740n-v6.o
```

Comment:
Yes, I compile it and test on router - all works fine.
But if this config wrong - I will fix it, compile and test again.
Thanks!